openqabot/utils.py

```diff
@@ -21,6 +22,20 @@ def create_logger(name: str) -> logging.Logger:
     return log
 
 
+def get_yml_list(path: Path) -> List[Path]:
+    """
+    Create a list of YML filesnames from a folder
+    or from a single file path.
+    """
+    yml_list = []
+    extension = "*.yml"
+    if path.is_file() and path.match(extension):
```

Comment:
```suggestion
    extensions = (".yml", ".yaml")
    if path.is_file() and path.name.endswith(extensions):
```

Reply:
I elaborated it a little bit more. `.something` is ok for `.endswith` but it is not for `.glob` that expect a "search pattern" like `*.something` with the `*`